docs/guidance/identifiers.md

```diff
@@ -0,0 +1,40 @@
+# Project identifiers
+
+A project identifier is a globally unique identifier for an infrastructure project. Every project in OC4IDS has a project identifier in the `id` field.
+
+Project identifiers can be used to join up data published at different times or from different systems; for example, including a project identifier in contracting data makes it possible to join up data on the design, construction and supervision contracts within a single infrastructure project.
+
+## Project identifiers in contracting data
+
+A common need is to access data about the contracting processes related to an infrastructure project. When contracting systems use consistent identifiers to refer to infrastructure projects, this becomes possible. An example use case is automatically checking which projects have related contracting data, and then manually filtering projects for further scrutiny, monitoring or data collection.
+
+There are different approaches to including project identifiers in contracting data, with the best solution depending on the context of an implementation:
+
+* **Include a free-text field for project identifiers in procurement systems** and work with each group that enters data to make sure this is populated according to some defined pattern.
```

Comment:
Change 'each group that enters data' to 'officials entering procurement information' to be consistent with next solution.

Reply:
Updated in https://github.com/open-contracting/infrastructure/pull/97/commits/acaa4b9fc84a4b1e1fba14ac3e30d85bda1a46f0